src/components/FlagGroupContent/FlagGroupContent.js

```diff
@@ -0,0 +1,122 @@
+import React, { useState } from 'react'
```

Comment:
reminder of how I like to organize imports

1: All third party libraries, alphabetical
2: All our code, alphabetical so components/ come before store/s
3. the .scss file last